src/lib/isMobilePhone.js

```diff
@@ -17,6 +17,7 @@ const phones = {
   'cs-CZ': /^(\+?420)? ?[1-9][0-9]{2} ?[0-9]{3} ?[0-9]{3}$/,
   'da-DK': /^(\+?45)?\s?\d{2}\s?\d{2}\s?\d{2}\s?\d{2}$/,
   'de-DE': /^(\+49)?0?1(5[0-25-9]\d|6([23]|0\d?)|7([0-57-9]|6\d))\d{7}$/,
+  'de-AT': /^(\+43)?0?6\d{9}$/,
```

Comment:
I am leaving some comment here based on my research. 
I come across these statements

```
Austrian mobile (Handy) telephone numbers begin with 0650 or higher up to 0699
(eg 0664 plus the rest of the number). Numbers beginning with 0900 are exorbitantly charged and best avoided. 
Some large organizations have ‘050’ numbers, which do not need an area code 
(a local call from a landline, but more expensive from a mobile phone). 
``` 
[Link](https://www.lonelyplanet.com/austria/telephone)
Again, I come to across below
```
 Austria’s country phone code is 43. 
There are no standard lengths for either area codes or subscribers’ numbers in Austria,
so some subscribers’ numbers are as short as three digits! 
For example, area phone codes can be anywhere from three digits (such as Linz’s 070) 
to five digits (such as Kitzbuehel’s 05356); 
the telephone numbers range from four to seven digits.
```
[Link for above](https://www.travelsignposts.com/Austria/useful-facts/find-an-austria-phone-number-or-phone-code)
I don't know if the statements above are valid. If valid, then it means regex will need some updates.